projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -74,6 +73,13 @@ $go-table-body-row-expanded-border: rgba(240, 240, 240, 0);
     &:last-of-type {
       border-top-right-radius: $global-radius;
     }
+
+    &--sticky {
+      @include z-index(2);
+      background: $theme-light-bg;
+      position: sticky;
```

Comment:
_technically_ this will not work in IE. I think we're okay with still using this though. Because IE is being deprecated, in this case the fallback is that the header will behave as it does in the default.